### DISCUSSION TITLE
Dual display not working. No start menu pops up.

### DISCUSSION THREAD

**Discussion by ekolpack** [valinet/ExplorerPatcher#3099] [Q&A]
Dell Precision 7770 Laptop w/ NVidia RTX A4500 GPU
Laptop screen set to 1080p 60hz (can't go any lower)
Second monitor set to 4k 30hz (can't go any higher)
Windows 11 Pro 23H2
Monitors are set to "Extend these displays" not duplicate

I had one taskbar/start menu on each display that worked properly (except for the stupid "recommended section" I couldn't get rid of). Installed latest EP to kill the recommended section. Now I can't get my start menu on the second display to pop up at all. The taskbar is there, and when I click Start, it seems like it TRIES to pop up (because the search box on the taskbar disappears) but no start menu appears on either the second display, or the primary laptop display. The primary laptop display taskbar functions as normal.

I've tried toggling tons of settings in both EP and Windows settings, restarting Explorer, restarting computer, turning off transparency, messing with scaling options on both screens, uninstalling/reinstalling EP. I've spent the past couple hours messing with settings and digging through menus, toggling stuff on an off.  I can't for the life of me get the start menu to pop up on my secondary monitor.

I've tried toggling "Show taskbar on multiple displays" on and off, as well as "When using multiple displays, show my taskbar apps on...." options. 

Please help! Thanks in advance!

*Edit: With custom scaling set to 175%, I can see a tiny portion of the start menu pop up on my LAPTOP screen, when I click the start menu on my 4K MONITOR! So it seems the menu is popping up, but on the wrong screen, and it doesn't look right, all I see is a portion of the search box. I tried to take a screen shot, but it's zoomed in, so you can't even see what I'm talking about. I may have to take a cell phone photo of my screen.

**Comment by ekolpack** [comment]
![Uploading TEMP_20240406_111651_HDR.jpg…]()
Okay, here's a physical photo of the glitch. You can see where I clicked the start bar on the 4K monitor (top of photo) but only a portion of the start menu appears on the OTHER display (my laptop) in the lower right corner. I can only see this when scaling is set high.

**Comment by ekolpack** [comment]
If I set the same resolution on both screens (1920x1080) then the start menu works, kinda. Clicking the start button on the second display causes it to open normally on the OTHER display. I'm going to uninstall EP now, because this is more of a PITA than the stupid wasted "recommended" space. But if anyone has any solutions, please reply back here and I may try again at a later date. Thanks again in advance!

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.
